.github/workflows/summon.yml

```diff
@@ -0,0 +1,68 @@
+# vim: set ft=yaml ts=2 sw=2:
+
+name: Summon Tasks
+
+on: [push]
+
+jobs:
+  lint:
+    strategy:
+      matrix:
+        python: ['3.7', '3.10']
+        os: [ubuntu-latest]
+
+    name: Static checks
+
+    runs-on: ${{ matrix.os }}
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up Python ${{ matrix.python }}
+        uses: actions/setup-python@v1
+        with:
+          python-version: ${{ matrix.python }}
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install poetry
+          poetry install
+
+      - name: Run static checks
+        run: poetry run summon static-checks
+
+
+  # test:
```

Comment:
Looks like the test part is commented here, but the readme says to execute `summon test`

Reply:
It's commented on the CI because tests don't work without a `config.json`. This is being worked on in another branch.

Reply:
Maybe let it just on that other branch then? So this one keeps "clean".